modules/govuk_elasticsearch/templates/es-backup-s3.erb

```diff
@@ -15,6 +15,19 @@ function nagios_passive () {
 
 trap nagios_passive EXIT
 
+<%- if scope.lookupvar('::aws_migration') %>
+  # Get the instance ID for the specific machine.
+  INSTANCE_ID=ec2metadata --instance-id
+  # Which instance ID is enabled for backups?
+  INSTANCE_WITH_BACKUPS_ENABLED=aws ec2 describe-tags --filters "Name=key,Values=backups_enabled" "Name=value,Values=1" | grep ResourceId | grep "i\-"
```

Comment:
The exec needs to be wrapped in $() or \` `

Instead of `grep`, you can use `jq` to parse the JSON result, same as we do in the user_data. It would look like:

```
INSTANCE_WITH_BACKUPS_ENABLED=$(aws ec2 describe-tags --filters "Name=key,Values=backups_enabled" "Name=value,Values=1" | jq -r '.Tags[] | [.ResourceId] | @csv' | tr -d '"' )
```

Reply:
That's clever! Thanks.